index.html

```diff
@@ -2825,125 +2827,265 @@ <h3>Working with personal names</h3>
+
+<div class="req" id="loc_names_relevant_info">
+<p class="advisement">If parts of a person's name are captured separately, ensure that the separate items can capture all relevant information.</p>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/International/questions/qa-personal-names#splitting">Strategies for splitting up names</a>, in <cite>Personal Names Around the World</cite>.</p>
 	</details>
+	<details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
 	</div>
 
-	<div class="req" id="loc_names_algorithms">
+<p>For example, don't assume that the order they will provide names in will be given followed by family.</p>
+
+<div class="req" id="loc_names_algorithms">
 	<p class="advisement">Be careful about assumptions built into algorithms that pull out the parts of a name automatically.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/International/questions/qa-personal-names#splitting">Strategies for splitting up names</a>, in <cite>Personal Names Around the World</cite>.</p>
 	</details>
-	</div>
+    <details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
+</div>
+
+<p>For example, the v-card and h-card approach of implied “n” optimization could have difficulties with, say, Chinese names. The input form should be as clear as possible when telling people how to specify their name, so that you capture the data you think you need.</p>
 
 	<div class="req" id="loc_names_initials">
 	<p class="advisement">Don't assume that a single letter name is an initial.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/International/questions/qa-personal-names#splitting">Strategies for splitting up names</a>, in <cite>Personal Names Around the World</cite>.</p>
 	</details>
+	<details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
 	</div>
 
+<p>People do have names that are one letter long. These people can have problems if the form validator refuses to accept their name and demands that they supply their name in full. If you want to encourage people not to use initials, perhaps you should make that a warning message, rather than block the form submission.</p>
```

Comment:
Maybe it doesn't need to be written here, but I have a similar and real example.

[Pengyou](https://en.wikipedia.org/wiki/Pengyou) requires usernames to be real names. A friend of mine tried to register with his real name, but the form validator says it's not a real name and refuses to accept his name, most likely because his family name is rare. So sites shouldn't assume people's family names are in a finite set.

Reply:
Look good to me. Do you want to add it in this PR or a separate issue/PR?

Reply:
I added a commit [3] to this PR.